gdi/get-data-in/application/java/instrumentation/java-servers-instructions.rst

```diff
@@ -139,6 +139,8 @@ Add the path to the JVM agent to your Tomcat or TomEE startup script:
 
             set CATALINA_OPTS=%CATALINA_OPTS% -javaagent:"<Drive>:\path\to\splunk-otel-javaagent.jar"
 
+.. note:: For Tomcat instances running as Windows Services, add the ``-javaagent`` argument to the list of Java Options using the Tomcat9w GUI application. 
```

Comment:
Can you reformat the note so that it appears like this, within the Windows tab:

```
   .. group-tab:: Windows

      On Windows, add the following line to the ``<tomcat_home>\bin\setenv.bat`` file:
   
         .. code-block:: shell

            set CATALINA_OPTS=%CATALINA_OPTS% -javaagent:"<Drive>:\path\to\splunk-otel-javaagent.jar"

      .. note:: 
         
         For Tomcat instances that run as a Windows service, use the Tomcat9w GUI to add the ``-javaagent`` argument to the list of Java Options. 
```

Reply:
@dmitchsplunk Please update the note with the following text:

```
When running Tomcat as a Windows service, the setenv.bat script is not used. To add the Java agent to your Tomcat Windows service, run the tomcat<version-number>w.exe utility as an administrator, then add the ``-javaagent`` argument to the list of :guilabel:`Java Options`.
```

Reply:
@pauljwil it's done, thanks.